components/edit-collective/sections/Host.js

```diff
@@ -32,6 +32,13 @@ const Fineprint = styled.div`
   font-size: 14px;
 `;
 
+const EditCollectiveHostSection = styled.div`
+  h2 label {
+    cursor: pointer;
+    width: auto;
+  }
```

Comment:
I don't think this has any effect. Also, as per your previous comment, what we want is to have a cursor effect on the select. So we should do:

```suggestion
  select {
    cursor: pointer;
  }
```

Reply:
@Betree : I think it does have an effect; since hovering over the labels such as `No Fiscal Host` and `Use own Fiscal Host` will change to pointer which happens due to that. 

Ah, I've somehow missed the obvious `select` css selector. Didn't know it was that simple. I just added it. 👍🏽